04/2_print_string_buffer/fixed_string.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "fixed_string.h"
+
+static int s_is_called;

```

Comment:
bad name.

s_buffer_empty is probably better name

Reply:
you probably don't need this variable.. instead u do need an index that keeps track of how many chars are in buffer instead